### ISSUE TITLE
Provide a start-paused solution

### ISSUE THREAD

**Issue by caprica** [caprica/vlcj#955]
For some use-cases, starting the media player paused on the first frame is desirable.

This is not difficult to implement, but it is not obvious.

It would be good to either:

1. document this somewhere (this issue, at least initially)
2. add a specific example
3. somehow build it into the vlcj media player (so it would be transparent to the library user)

For point 3 above, this could be similar to how there is already play vs start - play just plays, start plays and waits for a playing event.

The issue with start-paused is that it requires a specific media option to be passed - and we would have to handle the case where this is added in addition to any media options the caller provides themselves.

**Comment by caprica** [comment]
For options 3, could just tweak the existing MediaPlayerLatch used by the start() API to countdown the latch also on a paused event.

The caller would then pass "start-paused" with media options if that's what they want (they would have to use start() rather than play() in this case).

This would probably be very simple in fact.

**Comment by caprica** [comment]
It might be as simple as:
```
mediaPlayer.controls().stop();
mediaPlayer.media().start(mrl, "start-paused");
mediaPlayer.controls().setPosition(0);
```
Note that stop *is* required for the case where the video is already playing.

This stop is actually required because the MediaPlayerLatch has a conditional where it checks that the media player is not playing before processing the latch.

**Comment by caprica** [comment]
It might be just as well to leave this for library users to implement, which brings this issue back to at least documenting the solution.

**Comment by caprica** [comment]
Still not sure, it might be OK to provide a very simple wrapper around start() that implements the three operations above.

**Comment by caprica** [comment]
This can be cherry-picked for 4.x.

**Comment by caprica** [comment]
Resolved at 89b08faff7cbd9b48333a2a7c2473d11603c55e9.
